docu/files/ModuleExtensibility.html.md

```diff
@@ -114,12 +110,30 @@ BuilderExtensible.on('myPluginEvent', function(data) {
 
 ### Register host and plug-in modules
 
-Extensible hosts and plug-in modules, just like all other Builder modules, must be registered in YaaS.
-So, after you have implemented and deployed them, use the Builder to register the modules in YaaS and include them in the target packages that your project provides.   
+Extensible hosts and plug-in modules must be registered in YaaS just like all other Builder modules. So, after you have implemented and deployed them, use the Builder to register the modules in YaaS. Include the hosts and plug-in modules in the target packages that your project provides.   
 
 Defining a host module is no different from defining any other Builder module. 
 When registering a plug-in module, you should complete the same tasks as for other modules:
  - Provide a URL for the **module.json** file where it was deployed.
  - Specify what the plug-in can do on behalf of the tenant (the *authorization scopes* of the module),
 but pay special attention when defining redirect URIs.
  - Make sure that you define the redirect URI for the plug-in: `https://{baseURL of your plug-in}/builder/auth.html`. This endpoint is important for the plug-in to capture the access token that was granted for its authorization context.
+ 
+### Controlling slot size 
+
+Since plug-in modules do not all have the same size, it is important that you have the ability to adjust the size of the iframe in which the plug-in should fit. For example, a plug-in that displays product ratings may require more space than one that simply displays a single button. 
```

Comment:
Because all plug-in modules are not the same size, you need the ability to adjust the size of the iframe in which the plug-in fits. For example, a plug-in that displays product ratings might require more space than one that simply displays a single button.